contracts/lib/time_series/structs.cairo

```diff
@@ -0,0 +1,27 @@
+%lang starknet
+
+using NEVER = felt
+
+struct TickElem:
+    member tick : felt
```

Comment:
Why not call this `timestamp`?

Reply:
timestamp is a very specific representation, my thought was this may not always be a timestamp so there should be some flexibility in the general type.

https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.xticks.html was my thought process for the name.  Not married to the title, but didn't want to use something overly specific.